produce/lib/produce/commands_generator.rb

```diff
@@ -146,8 +146,11 @@ def run
         FastlaneCore::CommanderGenerator.new.generate(Produce::Options.available_options, command: c)
 
         c.action do |args, options|
-          allowed_keys = Produce::Options.available_options.collect(&:key)
-          Produce.config = FastlaneCore::Configuration.create(Produce::Options.available_options, options.__hash__.select { |key, value| allowed_keys.include? key })
+          extra_options = [FastlaneCore::ConfigItem.new(key: :merchant_name),FastlaneCore::ConfigItem.new(key: :merchant_identifier)]
```

Comment:
I saw that you removed these from the options file, just curious to know the reason for that? I think its always helpful to have all available options in the `<tool name>/options.rb` file as a sort of documentation.